scripts/zkai_d128_range_policy_discipline_gate.py

```
@@ -0,0 +1,777 @@
+    payload = {
+        "schema": SCHEMA,
+        "decision": DECISION,
+        "question": "Does the d128 statement-bound transformer block require per-tensor range policy rather than one global q8 semantic bound?",
+        "source_evidence_manifest": source_manifest(snapshots or source_snapshots()),
+        "range_policy_commitment": policy_commitment,
+        "tensor_policies": rows,
+        "summary": build_summary(rows) | {"range_policy_commitment": policy_commitment},
+        "non_claims": NON_CLAIMS,
+        "validation_commands": VALIDATION_COMMANDS,
+    }
+    return payload
```

Ruff (0.15.12)

[warning] Unnecessary assignment to `payload` before `return` statement

Remove unnecessary assignment

(RET504)